src/api/schemas.ts

```diff
@@ -7,3 +7,7 @@ export type SDKKeys = z.infer<typeof schemas.SDKKeys>
 export type Variable = z.infer<typeof schemas.Variable>
 export type Variation = z.infer<typeof schemas.Variation>
 export type Feature = z.infer<typeof schemas.Feature>
+export type CreateEnvironmentParams = z.infer<typeof schemas.CreateEnvironmentDto>
```

Comment:
what are these for if they're being replaced by the dto?

Reply:
the `CreateEnvironmentDto` in the schemas is a zodObject, so in order to use it as a typescript type we need to use `z.infer` to get the actual type of the parsed object. i tried a bunch of alternatives but couldn't find a better way to export both the type and the actual zodObject object that you can use for runtime validation, so i just have both here